geode-core/src/main/java/org/apache/geode/management/internal/cli/Launcher.java

```diff
@@ -99,7 +99,6 @@ protected Launcher() {
     this.allowedCommandLineCommands.add(CliStrings.START_JCONSOLE);
     this.allowedCommandLineCommands.add(CliStrings.START_JVISUALVM);
     this.allowedCommandLineCommands.add(CliStrings.START_LOCATOR);
-    this.allowedCommandLineCommands.add(CliStrings.START_MANAGER);
```

Comment:
Sure! Almost none of the `--- manager` CliStrings were used, so I deleted them. Only `start manager` was ever used, and only in this place, so I figured it would be safe to get rid of it. Do you think this might break any backwards compatibility / should I roll back the change?